File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/service/dtos/CreateServiceScheduleResponse.java
package com.slotify.backend.spring.service.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.time.DayOfWeek;
import java.util.UUID;

@Builder
@Getter
@ToString
public class CreateServiceScheduleResponse {

    private UUID scheduleId;
    private String serviceName;
    private Integer dayOfWeek;
    private String interval;
}

File: frontend/app/src/main/java/com/raj/slotify/dtos/service/CreateServiceScheduleResponse.kt
package com.raj.slotify.dtos.service

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class CreateServiceScheduleResponse(

    @SerializedName("scheduleId")
    val scheduleId: UUID,

    @SerializedName("serviceName")
    val serviceName: String,

    @SerializedName("dayOfWeek")
    val dayOfWeek: Int,

    @SerializedName("interval")
    val interval: String

)
